crates/gossip/proto/gossip.proto

```diff
@@ -90,6 +90,11 @@ message K2GossipInitiateMessage {
   // The DHT sectors covered by the union of the agents in the participating_agents list.
   ArcSetMessage arc_set = 11;
 
+  // A tie breaker so that conflicting initiates between a pair of peers can be resolved without a further message.
+  //
+  // This must be a random number, selected per gossip round.
+  uint32 tie_breaker = 12;
```

Comment:
The webrtc algorithm does something similar but without having to send an extra bit of data by using the lexical ordering of the peer id. Do you think that would work here?

Reply:
I did consider that. I was thinking that because of the asymmetry in the gossip, it'd be better to have a random choice. A peer can't strategically choose their peer id like they could choose their tie breaker but for honest peers, random is preferable I think